app/views/staff/activities/new.html.haml

```diff
@@ -0,0 +1,55 @@
+=content_for :page_title_prefix, t("page_title.activity.new")
+
+%p= link_to t("generic.link.back"), activities_path, class: "govuk-back-link"
+%main.govuk-main-wrapper#main-content{ role: "main" }
+  .govuk-grid-row
+    .govuk-grid-column-two-thirds
+      %h1.govuk-heading-xl
+        = t("page_title.activity.new")
+
+      = simple_form_for @activity do |f|
+        .govuk-form-group
+          = f.input :fund_id,
+                    wrapper: "select",
+                    collection: @funds
+          = f.input :identifier
+          = f.input :sector,
+                    wrapper: "select",
+                    collection: yaml_to_options("activity", "sector")
+          = f.input :title
+          = f.input :description
```

Comment:
This should be a textarea rather than the single rowed text input as we are asking for more information here and examples are good chunky paragraphs.